internal/proxy/oauthproxy_test.go

```diff
@@ -1871,84 +984,84 @@ func TestHTTPSRedirect(t *testing.T) {
 	}{
 		{
 			name:          "no https redirect with http and cookie_secure=false and authenticated=true",
-			url:           "http://foo.sso.dev/",
+			url:           "http://localhost/",
```

Comment:
I'm curious about the many `localhost` changes in these tests.  At a glance, they seem like a loss of info, but that's because I'm assuming that we must have some test cases where the distinction between `foo.sso.dev` and `bar.sso.dev` is important.

Reply:
We used to -- but not anymore. We've factored out the routing logic out of `OAuthProxy` and into the hostmux router. The `OAuthProxy` no longer does any routing so the changes between `foo.sso.dev` and `bar.soo.dev` are no longer important. I use `localhost` instead of these domains in response to https://github.com/buzzfeed/sso/issues/140 -- really they could be anything but `localhost` seemed safest.